activestorage/netcdf_to_zarr.py

```diff
@@ -7,11 +7,10 @@
 from kerchunk.hdf import SingleHdf5ToZarr
 
 
-def gen_json(file_url, fs, fs2, **so):
+def gen_json(file_url, fs, fs2, variable, **so):
```

Comment:
maybe use `varname` as below? Also it's unused at the moment?

Reply:
Sure - I'll change the name. It is used a few lines down ...

Reply:
cools, bud!